reladomogen/src/main/templates/CvsComment.jspi

```diff
@@ -1,26 +0,0 @@
-<%--
-Copyright 2016 Goldman Sachs.
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-  http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing,
-software distributed under the License is distributed on an
-"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-KIND, either express or implied.  See the License for the
-specific language governing permissions and limitations
-under the License.
---%>
-<% if (wrapper.isGenerateFileHeaders()){%>
```

Comment:
this is all based on a flag, so it' optional. For backward compatibility, we can leave this in, just turn the flag from the default of "true" to "false" in CoreMithraGenerator